<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulation</class>
 <widget class="QDialog" name="Simulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1044</width>
        <height>515</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,1">
         <item>
          <layout class="QGridLayout" name="elevatorLayout"/>
         </item>
         <item>
          <layout class="QGridLayout" name="passengerLayout"/>
         </item>
         <item>
          <layout class="QGridLayout" name="floorLayout"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="simulationInfoLayout">
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string>Status Label</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="timeLcdNumber"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
